### PR TITLE
Setup tailscale connection

### PR DESCRIPTION
**Pull request by GrigoriyMikhalkin**

### CODE REVIEW

**Comment by majst01** [comment]
@GrigoriyMikhalkin #142 is merged

File: debian/docker-make.debian.yaml [hunk 1]
```diff
@@ -25,7 +25,7 @@ builds:
       - BASE_OS_VERSION=10
       - DOCKER_APT_CHANNEL=buster
       - FRR_VERSION=frr-8
-      - FRR_VERSION_DETAIL=8.2.2-0~deb10u1
+      - FRR_VERSION_DETAIL=8.3-0~deb10u1
```
Comment (majst01) [inline debian/docker-make.debian.yaml:28]: please do not update frr with this PR, this must be done in a separate one

**Comment by GrigoriyMikhalkin** [comment]
@majst01 Firewall image need to have `iptables` binary for `tailscaled` to work. There's [ongoing work to migrate `tailscaled` from using `iptables` to `netlink`](https://github.com/tailscale/tailscale/issues/391), but at the moment it still relies on it.

**Comment by majst01** [comment]
> @majst01 Firewall image need to have `iptables` binary for `tailscaled` to work. There's [ongoing work to migrate `tailscaled` from using `iptables` to `netlink`](https://github.com/tailscale/tailscale/issues/391), but at the moment it still relies on it.

This is bad, iptables will interfere with nftables i think, but for the start we should at least try with both installed.

File: firewall/Dockerfile [hunk 2]
```diff
@@ -20,7 +23,8 @@ RUN apt-get update --quiet \
     fever \
     tcpdump \
  && apt-get remove --yes iptables \
- && apt --yes autoremove
+ && apt --yes autoremove \
+ && apt install --yes iptables
```
Comment (majst01) [inline firewall/Dockerfile:27]: just dont remote iptables, add it to the list of packages to be installed and remove the apt-get remove iptables step

**Review by majst01 — CHANGES_REQUESTED** [review]

**Comment by GrigoriyMikhalkin** [comment]
@majst01 Update tiptables installation. Tested with mini-lab, it worked. So at least in mini-lab environment it's required to reinstall `iptables`.

**Comment by majst01** [comment]
> @majst01 Update `iptables` installation. Tested with mini-lab, it worked. So at least in mini-lab environment it's required to reinstall `iptables`.

Strange but good to know. Dont understand, can you please add a comment alongside these lines for later reference, otherwise someone will simply "optimize" this particular part.

**Review by majst01 — APPROVED** [review]

**Pull request merged**